tcms/settings/product.py

```diff
@@ -1,4 +1,6 @@
-# Django settings for product env.
+"""
```

Comment:
For this file disable `wildcard-import` and `unused-wildcard-import`. Add the disable pragma at the top of the file before docstring.